acquire/acquire.py

```diff
@@ -683,7 +681,7 @@ def _run(cls, target: Target, cli_args: argparse.Namespace, collector: Collector
 
         patterns = ["$Recycle.bin/*/$I*", "Recycler/*/INFO2", "Recycled/INFO2"]
 
-        if not cli_args.no_data_files:
+        if cli_args.data_files is None or cli_args.data_files:
```

Comment:
I assume `data_files` will be `None` if no argument is given? So that the default behaviour stayed the same (collect data files)?